### PR TITLE
Think6.5v2 keymap with working caps-lock indicator

### PR DESCRIPTION
**Pull request by dangjoeltang**
## Description

Custom keymap with working caps-lock indicator light, when using static RGB mode. The firmware that ships with the Think6.5v2 has a working Caps-lock indicator that lights up the top part of the badge. However, the version in `gray_studio/think65/solder` does not. 

This is a stopgap for those who re-flashed their board's and would like to have the indicator light functionality back. 

I will continue working to make a better default keymap and submit a keyboard update in a later PR.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dangjoeltang** [comment]
@mechmerlin, can you take a look at my keymap? I saw that you were the maintainer for the Think6.5v1 firmware

File: keyboards/gray_studio/think65/solder/keymaps/dangjoeltang/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 Qiaowei Tang
```
Comment (drashna) [inline keyboards/gray_studio/think65/solder/keymaps/dangjoeltang/config.h:1]: This file should be removed.

File: keyboards/gray_studio/think65/solder/keymaps/dangjoeltang/keymap.c [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define XXXXXXX KC_NO
```
Comment (drashna) [inline keyboards/gray_studio/think65/solder/keymaps/dangjoeltang/keymap.c:18]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/gray_studio/think65/solder/keymaps/dangjoeltang/rules.mk [hunk 3]
```diff
@@ -0,0 +1 @@
+# Build Options
```
Comment (fauxpark) [inline keyboards/gray_studio/think65/solder/keymaps/dangjoeltang/rules.mk:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**